settings/base.py

```diff
@@ -20,25 +25,32 @@ def path(*args):
 )
 
 MANAGERS = ADMINS
+# avoids a warning from django
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL. See travis.py in this directory
 # for an example if you'd like to run MySQL locally, and add that to your
 # local.py.
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'basket.db',
-    }
+    'default': config('DATABASE_URL',
+                      default='sqlite:///basket.db',
+                      cast=dj_database_url.parse),
 }
+if DATABASES['default']['ENGINE'] == 'django.db.backends.mysql':
+    DATABASES['default']['OPTIONS'] = {'init_command': 'SET storage_engine=InnoDB'}
 
-ALLOWED_HOSTS = [
-    '.allizom.org',
-    'basket.mozilla.com',
-    'basket.mozilla.org',
-]
-SESSION_COOKIE_SECURE = True
-CSRF_COOKIE_SECURE = True
+CACHES = {
+    'default': config('CACHE_URL',
+                      default='locmem://',
+                      cast=django_cache_url.parse)
+}
+
+ALLOWED_HOSTS = config('ALLOWED_HOSTS',
+                       default='.allizom.org, basket.mozilla.com, basket.mozilla.org',
+                       cast=Csv())
+SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', True, cast=bool)
+CSRF_COOKIE_SECURE = config('CSRF_COOKIE_SECURE', True, cast=bool)
```

Comment:
After looking at a big block of defaults lower down, I changed my mind. I prefer your way now :)